packages/mwp-app-render/src/components/PageWrap/index.jsx

```diff
@@ -41,6 +43,21 @@ class PageWrap extends React.Component {
 					now - window.performance.timing.navigationStart // this is the event that NR uses as 'start' of page load
 				);
 		}
+
+		// Add W3C UserTiming mark for TTI and measure (from navigationStart to newly created mark)
+		if (
+			window.performance &&
+			window.performance.mark &&
+			window.performance.measure
+		) {
+			window.performance.mark('meetup-tti');
+			window.performance.measure('meetup-tti', 'navigationStart', 'meetup-tti');
+		}
+
+		// Specially for Developer Tools in the browsers (Chrome & Firefox), create entry for the event so it shows up on Performance timeline
+		if (console && console.timeStamp) {
```

Comment:
no need to change, but we should be able to rely on `console` being globally available everywhere.

Reply:
Got it.